pc/rtp_transport.cc

```diff
@@ -245,6 +245,11 @@ void RtpTransport::OnReadPacket(rtc::PacketTransportInternal* transport,
                                 int flags) {
   TRACE_EVENT0("webrtc", "RtpTransport::OnReadPacket");
 
+  if (!incoming_rtp_enabled_) {
+    RTC_LOG(LS_WARNING) << "Dropping RTP packet because incoming RTP is disabled!";
```

Comment:
with a LS_WARNING for every packet, this could be quite noisy.

Reply:
It's only for every packet that we shouldn't receive in the first place.

Reply:
I suppose we'll get one hangup data message in some logs.  Other than that, we shouldn't ever get this line being hit unless someone is broken or doing an attack.